### DISCUSSION TITLE
Define cookie prefix in one location

### DISCUSSION THREAD

**Discussion by dalelotts** [epicweb-dev/epic-stack#867] [Ideas]
All cookies have a `en-` prefix, for epic-notes. 

Consider allowing the user to define the prefix 

When I implemented cookie consent I used the cookie category as the prefix so it was trivial to expire that category of cookie when consent was revoked.

**Comment by kentcdodds** [comment]
Hi @dalelotts,

How would you see that working? Would it be a question we ask during initial setup or a config-based thing? I really would prefer to avoid config-based things for the same reasons mentioned here: https://github.com/epicweb-dev/epic-stack/pull/526#issuecomment-1821674207

**Reply by dalelotts** [reply]
I think having vite statically replace the values at build time would work - see https://vitejs.dev/guide/env-and-mode.html
